tests/unit/test_zigzag.py

```diff
@@ -169,6 +170,7 @@ def test_missing_testcase_properties_xml(self, missing_testcase_properties_xml,
             zz = ZigZag(missing_testcase_properties_xml, simple_json_config, TOKEN)
             zz.parse()
 
+    @pytest.mark.skip(reason="The error that will be raised will be different")
```

Comment:
Is there a tech debt ticket to revisit this later?

Reply:
https://rpc-openstack.atlassian.net/browse/ASC-1558